### PR TITLE
[Feature] Introduce Layout

### PR DESCRIPTION
**Pull request by labbomb**
### Purpose

Please refer to the main `issue`  [#40](https://github.com/apache/incubator-paimon-webui/issues/40).
image

### CODE REVIEW

File: paimon-web-ui-new/src/layouts/content/components/menubar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          {
+            label: i18n.global.t('layout.playground'),
+            key: 'playground',
+          },
+          {
+            label: i18n.global.t('layout.metadata'),
+            key: 'metadata',
+          },
+          {
+            label: i18n.global.t('layout.CDC_Ingestion'),
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/layouts/content/components/menubar/index.tsx:41]:
```suggestion
            label: i18n.global.t('layout.cdc_ingestion'),
```

File: paimon-web-ui-new/src/layouts/content/components/menubar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            key: 'CDC_Ingestion',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/layouts/content/components/menubar/index.tsx:42]:
```suggestion
            key: 'cdc_ingestion',
```

**Review by SteNicholas — CHANGES_REQUESTED** [review]
@labbomb, thanks for contribution. I left minor comments for this change. PTAL.

File: paimon-web-ui-new/src/locales/zh/modules/layout.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+under the License. */
+
+export default {
+  playground: '查询控制台',
+  metadata: '元数据管理',
+  CDC_Ingestion: 'CDC 集成',
+  system: '系统管理',
+  light: '浅色',
+  dark: '暗色'
+}
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/locales/zh/modules/layout.ts:25]: New line here.

File: paimon-web-ui-new/src/api/request.ts [hunk 4]
```diff
@@ -45,12 +45,12 @@ axle.axios.interceptors.request.use(
 
 axle.axios.interceptors.response.use(
   (response) => {
-    const { code, message } = response.data
+    const { code, msg } = response.data
```
[the remark this answers is not in the document]
Comment (labbomb) [reply paimon-web-ui-new/src/api/request.ts:48]: Because the field returned by the interface is msg

**Review by SteNicholas — APPROVED** [review]
LGTM.

**Pull request merged**